internal/service/privatelinkendpointservicedatafederationonlinearchive/data_source_privatelink_endpoint_service_data_federation_online_archive_test.go

```diff
@@ -15,7 +15,7 @@ var (
 
 func TestAccNetworkPrivatelinkEndpointServiceDataFederationOnlineArchiveDS_basic(t *testing.T) {
 	var (
-		projectID               = acc.ProjectIDGlobal(t)
+		projectID               = acc.ProjectIDExecution(t)
```

Comment:
all functions like this will start with ProjectID,    we have ProjectIDExecution, ProjectIDGlobal and might have ProjectIDTrigger, etc.